designer/client/components/Menu/Menu.tsx

```diff
@@ -123,13 +123,13 @@ export default function Menu({ updateDownloadedAt, id }: Props) {
       )}
       {outputs.isVisible && (
         <Flyout title="Edit Outputs" onHide={outputs.hide} width="xlarge">
-          <OutputsEdit data={data} onCreate={outputs.hide} />
+          <OutputsEdit data={data} />
```

Comment:
Ah, this should be using context instead of the prop but I'll open a new issue for it